dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -118,6 +118,12 @@
 			label = "SCFG";
 		};
 
+		bbram: bb-ram@400af000 {
+			compatible = "nuvoton,npcx-bbram";
+			reg = <0x400af000 0x80>;
```

Comment:
The range doesn't cover the status register. Don't you cover it together?

Reply:
You're right, sorry, been working on a more robust version of this that also includes a driver to check the status. Will push the changes as soon as I finish testing them.